tools/qontract_cli.py

```diff
@@ -632,6 +632,26 @@ def bot_login(ctx, cluster_name):
     print(f"oc login --server {server} --token {token}")
 
 
+@get.command()
+@click.argument("org_name")
```

Comment:
Can you add documentation to this CLI method in the printed help? What does it do, what is `org_name`, etc...

Reply:
https://github.com/app-sre/qontract-reconcile/pull/2905/commits/c4f6cd60f57e9a8cf13148941c48c448f2f93550